packages/database/scripts/create-database.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+read -p "What is the mysql docker container name? [mysql]: " container_name
+container_name=${container_name:-mysql}
+
+read -p "What is the new database name? [datahub_development]: " database_name
+database_name=${database_name:-datahub_development}
+
+read -p "What is the mysql username? [root]: " user_name
+user_name=${user_name:-root}
+
+read -p "What is the mysql password?: " password
+
+docker exec ${container_name} mysql -u ${user_name} -p${password} -e "CREATE DATABASE ${database_name};" && 
```

Comment:
Should there be a space between `-p` and `${password}`?

Reply:
No, it should be without a space. I changed it to `--user=${user_name} --password=${password}`.